accessibility-checker-engine/help/WCAG21_Label_Accessible.mdx

```diff
@@ -27,7 +27,7 @@ Speech-input users navigate to an input field by speaking the visible field labe
 
 ### What to do
 
-* Provide an [accessible name value](https://www.w3.org/TR/wai-aria-1.2/#namecalculation) that matches or contains the visible label for this control. For example:
+* Provide an [accessible name value](https://w3c.github.io/aria-practices/#namecalculation) that matches or contains the visible label for this control. For example:
```

Comment:
I don't believe name calculation should be from the editor's draft...

Reply:
agree. These one shoud not have been changed. correcting back